lib/bot.js

```diff
@@ -28,9 +28,8 @@ const TwitchBot = class TwitchBot extends EventEmitter {
     this.username = username
     this.oauth = oauth
     this.channels = channels.map((channel) => {
-      channel = channel.toLowerCase()
-      return channel.charAt(0) !== '#' ? '#' + channel : channel
-    })
+      return parser.formatCHANNEL(channel);
+    });
```

Comment:
should `channels` in the constructor be a required param given that this would support joins and parts after initialization?

Reply:
Good call. Making it an optional parameter might be a good idea.